examples/ruby/sonar-project.properties

```diff
@@ -2,4 +2,4 @@ sonar.projectKey=twilio_twilio-oai-generator-ruby
 sonar.organization=twilio
```

Comment:
update and delete call usecase are missing. Can we have them of any of the specs